lib/rest.js

```diff
@@ -47,31 +46,22 @@
 		}
 
 		this.url = urllib.parse(url);
-		var boundary = 'JWBBV0Q07j8njeiL';
 
 		if (this.options.multipart) {
-			headers['content-type'] = 'multipart/form-data; boundary='+boundary;
-			payload = '';
+			var form = new FormData();
 
 			_.each(this.options.data, function(value, inputName) {
-				payload += '--'+boundary+'\r\n'+
-				'Content-Disposition: form-data;';
-				if(inputName == 'file_path' && fs.existsSync(value)) {
-					var parts = value.split(/\/|\\/);
+				if(inputName === 'file_path' && fs.existsSync(value)) {
+					var parts = value.split(/[\/\\]/);
 					var fileName = parts[parts.length - 1];
-
-					payload += 'name="file"; filename="'+fileName.replace(/"/g, '\"')+'"\r\n';
-					payload += 'Content-Type: '+mime.lookup(value);
-
-					value = fs.readFileSync(value);
+					form.append('file', fs.readFileSync(value), fileName);
 				} else {
-					payload += ' name="'+inputName.replace(/"/g, '\"')+'"';
+					form.append(inputName, value);
 				}
-
-				payload += '\r\n\r\n'+value+'\r\n';
 			});
-			payload += '--'+boundary+'--';
 
+            headers = form.getHeaders(headers);
```

Comment:
Indentation

Reply:
@tot-ra My bad, this project uses tabs and I didn't notice that.